<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>691</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_pathR"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_pathL"/>
     </item>
     <item row="1" column="1">
      <widget class="QListView" name="listView_right"/>
     </item>
     <item row="1" column="0">
      <widget class="QListView" name="listView_left"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
